doxygen/examples/tables/propertyLists.dox

```diff
@@ -179,11 +171,19 @@ creation property list.</td>
 <td>Sets/retrieves metadata cache and raw data chunk cache parameters.</td>
```

Comment:
Shouldn't this be Sets/Retrieves?  Same for Sets/gets.
Also, some are Set/get instead of Sets/gets.

Reply:
Didn't mean correct, right?